# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/lvm2/hotfix.patch.musl
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- LVM2.2.03.13/lib/device/dev-cache.c.vanilla	2021-08-30 17:09:00.142512466 +0200
+++ LVM2.2.03.13/lib/device/dev-cache.c	2021-08-30 17:15:37.044469955 +0200
@@ -29,6 +29,7 @@
 #include <unistd.h>
 #include <dirent.h>
 #include <locale.h>
+#include <sys/file.h>
 
 struct dev_iter {
 struct btree_iter *current;
--- LVM2.2.03.13/tools/lvmdevices.c.vanilla	2021-08-30 17:09:20.394510297 +0200
+++ LVM2.2.03.13/tools/lvmdevices.c	2021-08-30 17:15:45.733469024 +0200
@@ -16,6 +16,7 @@
 #include "lib/cache/lvmcache.h"
 #include "lib/device/device_id.h"
 #include "lib/device/dev-type.h"
+#include <sys/file.h>
 
 static void _search_devs_for_pvids(struct cmd_context *cmd, struct dm_list *search_pvids, struct dm_list *found_devs)
 {
